OSDOCS#16913: CQA for getting started with the CLI (#113944)
openshift/openshift-docs: modules/cli-logging-in.adoc

[id="cli-logging-in_{context}"]
= Logging in to the OpenShift CLI

[role="_abstract"]

> You can log in to the OpenShift CLI (`oc`) to access and manage your cluster.

Reviewer: ```suggestion
You can log in to the {oc-first} to access and manage your cluster.
```

Authentication headers are sent only when using HTTPS transport.
====

.Prerequisites

* You must have access to a {product-title} cluster.
* The {oc-first} is installed.

.Procedure

> . Enter the `oc login` command and pass in a user name:

Reviewer: ```suggestion
. Enter the `oc login` command and pass in a user name by running the following command:
```
this whole section lacks the language for accessibiilty; maybe add a bug to the spreadsheet to fix later if it feels out of scope now?

Author: Yeah, I don't disagree with a lot of your suggestions here, but we're supposed to keep our CQA PRs as minimal as possible, so I didn't make a lot of improvements. I'll see if there are any critical and will apply those, but might leave some alone.

> .Example output

Reviewer: we are entering output?

Author: Leaving this as is since I didn't touch it. And since it is technically output, and they do interact with it, but the values used are just examples, so I think it's fine.

Welcome! See 'oc help' to get started.
----
+
--

> ** For the `Server` prompt, enter the {product-title} server URL.

Reviewer: ```suggestion
* For the `Server` prompt, enter the {product-title} server URL.
```
just one bullet on these?

Author: No, technically the bullets are under step 2, so second level bullets are appropriate here. We have it that way in our callout alternative examples as well.

** For the `Use insecure connections?` prompt, enter whether to use insecure connections.
** For the `Username` prompt, enter the username to log in with.
** For the `Password` prompt, enter the user's password.
--
+
[NOTE]
====
If you are logged in to the web console, you can generate an `oc login` command that includes your token and server information. You can use the command to log in to the

> OpenShift CLI (`oc`) without the interactive prompts. To generate the command, select *Copy login command* from the username drop-down menu at the top right of the web console.

Reviewer: is there a hard return here? i don't see the error, but the line wrapping looks odd

Author: Yeah looks like a hard wrap, but since I didn't touch this line and it's not affecting the output, I'm going to leave it alone.

> OpenShift CLI (`oc`) without the interactive prompts. To generate the command, select *Copy login command* from the username drop-down menu at the top right of the web console.

Reviewer: ```suggestion
`oc` without the interactive prompts. To generate the command, select *Copy login command* from the username drop-down menu at the top right of the web console.
```

====
+

> You can now create a project or issue other commands for managing your cluster.

Reviewer: ```suggestion
. You can now create a project or issue other commands for managing your cluster.
```
I feel like this should be a step or a .Next steps entry

Author: It's really just a wrap-up statement. If anything, maybe a .Results section would be appropriate, but I think that's overkill for this.
